### ISSUE TITLE
[EN] [Core] Rich/Small Thorium Veins spawn rate

### ISSUE THREAD

**Issue by Propolys** [Atlantiss/BugTracker#1472]
Most of the zones supposed to have plenty of Thorium veins simply don't.

Notice on wowhead how the abundance is obvious:
http://www.wowhead.com/object=324
http://www.wowhead.com/object=175404

Also, on http://www.wowhead.com/item=10620/thorium-ore#comments , the comment clearly states:
"Silithus.....Thorium Veins all over the place....Above as well as under ground.

1 trip around the WHOLE map = 272 Ore, 10 Arcane Crystall & 8 other Gem's and enough Dense Stone to make 71 Dense Grinding Stones

Ive been there in my young days, but now the density is trippled....." (patch 4.0.6)

After a trip of mine in Silithus I barely found 2 Thorium Veins,

**Comment by Tj20** [comment]
I don't have hard numbers, but I also found farming Thorium very hard copared to all the other ores. Getting my mining skill from 250 to 275 (so I can farm in Outland) with green Thorium was a major pain in the rear.

Silithus and Un'goro had very few Thorium nodes (or somebody was there before me). Winterspring had plenty on my first round, but the respawn timer seems to be very long... 15-30 minutes I'd say. (I continued to fly rounds in Winterspring but found really nothing for the next 15 minutes. Then I did a dungeon, and after that the mining nodes had respawned, so I'd say 30 minutes after my first round)

Other ores have really high respawn rates (often they respawn nearly instantly, and Mithril in Un'Goro you even find 2 nodes directly next to each other most of the time), which seems to be *too* good... however Thorium is a real pain to farm for a non-rare ore.

**Comment by Satyr87** [comment]
i agree theres something wrong with thorium veins , there are few possibilities
1 respawn time is too long
2 the amount of veins is much lower than it should be
3 there are always 20 invisible people running around you and mining thorium before you get close enough to see it

**Comment by Tj20** [comment]
I can't speak for the Horde, but when I was farming ores, I did a /who Winterspring and found that I was the only Alliance player in Winterspring at that time.

**Comment by tmpone** [comment]
I've been two times in Winterspring as Alliance player - almost impossible to find Thorium Veins there...

**Comment by Pieczyyea** [comment]
I've been many times in Winterspring. I'm in Horde. Found 2 Veins.

**Comment by Haz3L** [comment]
Same problem is with cobalt veins.

**Comment by Propolys** [comment]
Cobalt goes a bit better though than Thorium. Still under the normal abundance.

**Comment by Satyr87** [comment]
ive made some research:
in case of thorium in winterspring it seems like the problem is very long respawn time , ive made a huge trip around the mountains when no one was playing and managed to mine few stacks
in case of cobalt in borean tundra and howling fjord seems like those veins are removed , i was flying around looking for it and literally made all quests in these zones when no one else was around 
(playing between 3-7am server time and checking /who) and there were like 5-10 veins on the whole map (similar thing with zul'drak but i just flew there 3 times across whole the map , no questing around)

**Comment by micha666** [comment]
I join the party. Today did a trip around winterspring found 2 nodes and 2 bugged nodes in ground(Owl wind thicket).

**Comment by TranceAccount** [comment]
I can say that there might be only a few Rich Thorium Veins available for mining.
There are enough Small ones.
